src/makeCalculator.js

```diff
@@ -38,6 +38,29 @@
  */
 function makeCalculator() {
   // write code here
+  let result = {
+    result: 0,
+    add: (value) => {
+      this.result += value;
+    },
+    subtract: (value) => {
+      this.result -= value;
+    },
+    multiply: (value) => {
+      this.result *= value;
+    },
+    divide: (value) => {
+      this.result /= value;
+    },
```

Comment:
You can add validation for zero-dividing

Reply:
dividing by zero not describe i task conditions
(so I dont know what I must do in this case: return result without any action or something else)